_IBC/noble-persistence.json

```diff
@@ -2,22 +2,22 @@
   "$schema": "../ibc_data.schema.json",
   "chain_1": {
     "chain_name": "noble",
-    "client_id": "07-tendermint-37",
-    "connection_id": "connection-41"
+    "client_id": "07-tendermint-61",
+    "connection_id": "connection-59"
   },
   "chain_2": {
     "chain_name": "persistence",
-    "client_id": "07-tendermint-154",
-    "connection_id": "connection-192"
+    "client_id": "07-tendermint-160",
+    "connection_id": "connection-198"
   },
   "channels": [
     {
       "chain_1": {
-        "channel_id": "channel-22",
+        "channel_id": "channel-36",
```

Comment:
can I ask why the old connection wasn't revived? Have governance voted to use a new connection? Or can we show that no token will be abandoned on the old channel?

Reply:
We counted it, it was < 5~10USDC.. and it was all team wallets to test, and 
we wanted the channel alive before governance voting period